src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -498,3 +499,45 @@ def load_arguments(self, _):
     with self.argument_context('functionapp deployment slot swap') as c:
         c.argument('action', help="swap types. use 'preview' to apply target slot's settings on the source slot first; use 'swap' to complete it; use 'reset' to reset the swap",
                    arg_type=get_enum_type(['swap', 'preview', 'reset']))
+
+
+# Access Resction Commands
+    with self.argument_context('webapp config access-restriction show') as c:
+        c.argument('name', arg_type=webapp_name_arg_type)
+    with self.argument_context('webapp config access-restriction add') as c:
+        c.argument('name', arg_type=webapp_name_arg_type)
+        c.argument('rule_name', options_list=['--rule-name', '-r'],
+                   help='Name of the access restriction rule to add')
+        c.argument('priority', options_list=['--priority', '-p'],
+                   help="Priority of the access restriction rule")
+        c.argument('description', options_list=['--description'],
```

Comment:
i suggest you remove the `options_list=['--description']` since you are not making any changes on the default one.

Reply:
Makes sense - I will optimize the arguments list

Reply:
@madsd as per the PMs we should add the preview tag for this feature. Thanks for pointing this out Yugang.

Reply:
Preview tag added